src/common-utils/install.sh

```diff
@@ -140,7 +140,12 @@ if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
     if [[ ! -z $(apt-cache --names-only search ^libssl1.1$) ]]; then
         package_list="${package_list}       libssl1.1"
     fi
-    
+
+    # Install libssl3 if available
+    if [[ ! -z $(apt-cache --names-only search ^libssl3$) ]]; then
```

Comment:
Can we bump the minor version for `common-utils` Feature for the changes to take effect?

Reply:
Fixed!